src/components/organisms/UiList/stories/ListBox.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <UiList
+    v-bind="args"
+    class="list-box"
+  />
+  <div class="search-dropdown-input-row">
+    <UiButton
```

Comment:
I thought that this button would be in folder _internal as ready to use component

Reply:
In design padding are different than in your button:
design:
<img width="559" alt="image" src="https://github.com/infermedica/component-library/assets/37744371/e2577259-58bf-4c5f-a4ee-794539a305df">

storybook:
<img width="1223" alt="image" src="https://github.com/infermedica/component-library/assets/37744371/3806004f-0697-4468-b202-9f77284af22c">

Reply:
Button also should have state hover like in design:
<img width="452" alt="image" src="https://github.com/infermedica/component-library/assets/37744371/8d3bd5ae-8be5-4593-84db-261816d9c89d">

Currently, it doesn't have it.
Also, whole row in the design is a button, not only the text `Didn't find chronic condition?`